src/views/static/tools/tools-view.js

```diff
@@ -0,0 +1,22 @@
+import { customElement, html, LitElement } from 'lit-element';
```

Comment:
This page can probably go too

Reply:
I opt to keep it. Its a template for wider functioanlity to be constructed in the (near) future.

Reply:
oh wait, this is under static. no that is wrong. we can indeed delete it. I will take care of this.